FelipeMaxRosa/react-template
# feat: add js minification

### Description

Improved minification of JS using [terser](https://webpack.js.org/plugins/terser-webpack-plugin/) and apply updates on webpack configurations.

#### Screenshots

<img width="1036" alt="image" src="https://user-images.githubusercontent.com/36519478/162886354-61148bc7-fd74-4d4f-a03f-f568265fce32.png">

Please note file size of final js

#### Type of change

Please delete options that are not relevant.


### How to test it

1. Open terminal
2. Change to main branch
3. run "yarn build"
4. inspect output size and notice all is building as expected, maybe run "yarn server"
5. Change to this branch
6. run "yarn build"
7. inspect output size and notice all is building as expected, maybe run "yarn server"
8. should notice improvement in file size

### Checklist:

- [x] The branch is updated with the last main branch changes.
